internal/build/job/maker.go

```diff
@@ -130,7 +173,8 @@ func MakeSecretVolumes(secretRefs []v1.LocalObjectReference) []v1.Volume {
 	return volumes
 }
 
-func MakeSecretVolumeMounts(secretRefs []v1.LocalObjectReference) []v1.VolumeMount {
+func (m *maker) makeBuildSecretVolumeMounts(secretRefs []v1.LocalObjectReference) []v1.VolumeMount {
```

Comment:
I don't think this needs to be a method.

Reply:
Agree. Fixing.